bleach/__init__.py

```diff
@@ -43,6 +44,8 @@
 
 ALLOWED_STYLES = []
 
+ALLOWED_PROTOCOLS = copy.copy(HTMLSanitizer.acceptable_protocols)
```

Comment:
I'm guessing based on seeing a lot of user generated posts on our platform but: ftp, rtsp, nntp, webcal, feed.

We tend to look in the version history on PyPI so that would be ideal for us, but a CHANGES file is ok too, just a bit harder to stumble over when you're doing a big "lets update some libraries" drive.